src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/request/CaseWorkersProfileCreationRequest.java

```diff
@@ -34,7 +34,7 @@ public CaseWorkersProfileCreationRequest(@JsonProperty("first_name") String firs
             @JsonProperty("region_id") Integer regionId,
             @JsonProperty("user_type") String userType,
             @JsonProperty("region") String region,
-            @JsonProperty("delete_flag") boolean deleteFlag,
+            @JsonProperty("suspeded") boolean suspended,
             @JsonProperty("idam_roles") Set<String> idamRoles,
```

Comment:
there seems to be a type  as the jspn property is defined as. -" suspeded"

Reply:
Good spot! Will amend this now